### PR TITLE
Change: swift 4.1 min

### PR DESCRIPTION
**Pull request by aliak00**

### CODE REVIEW

File: Source/Manager/Identifiers/EmailAddress.swift [hunk 1]
```diff
@@ -54,3 +54,9 @@ public struct EmailAddress: IdentifierProtocol {
         return identifier
     }
 }
+
+extension EmailAddress: Equatable {
+    public static func == (lhs: EmailAddress, rhs: EmailAddress) -> Bool {
+        return lhs.normalizedString == rhs.normalizedString
+    }
+}
```
Comment (fespinoza) [inline Source/Manager/Identifiers/EmailAddress.swift]: do you need this?

it seems the only property of `EmailAddress` is `emailAddress` which is a string that is already `Equatable`, then you just declare `EmailAddress: Equatable` you don't need to define `==` then

File: Tests/Core/JWTTests.swift [hunk 2]
```diff
@@ -53,7 +53,7 @@ class JWTTests: QuickSpec {
                 expect { try proxy.toJSON(string: "blah.bl{ah.blah") }.to(throwError(JWTHelperError.componentDecodingError("")))
             }
 
-            fit("Should fail on invalid json in second jwt component") {
```
Comment (fespinoza) [inline Tests/Core/JWTTests.swift]: what is fit?

File: Source/Manager/Identifiers/EmailAddress.swift (hunk 1 above)
Comment (aliak00) [reply Source/Manager/Identifiers/EmailAddress.swift]: Ah nice catch! Thanks!

File: Tests/Core/JWTTests.swift (hunk 2 above)
Comment (aliak00) [reply Tests/Core/JWTTests.swift]: It makes "only this test" run :p somehow got in from a previous commit.

**Pull request merged**